components/molecule/notification/src/index.js

```diff
@@ -158,6 +162,9 @@ const MoleculeNotification = memo(
 MoleculeNotification.displayName = 'MoleculeNotification'
 
 MoleculeNotification.propTypes = {
+  /** Vertical alignment for content */
+  alignContent: PropTypes.oneOf(Object.keys(ALIGN_ITEMS)),
```

Comment:
I'm wondering... isn't it a bit confusing to name this prop `alignContent` and not `alignItems`?

Reply:
a simple demo Article would we helpful 4 testing

Reply:
basically because `align-content` is a CSS property name too...